src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,167 @@
+import { styled, useTheme } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import Drawer from '@mui/material/Drawer';
+import CssBaseline from '@mui/material/CssBaseline';
+import MuiAppBar, { AppBarProps as MuiAppBarProps } from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import List from '@mui/material/List';
+import Typography from '@mui/material/Typography';
+import Divider from '@mui/material/Divider';
+import IconButton from '@mui/material/IconButton';
+import MenuIcon from "@material-ui/icons/Menu";
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import { ChevronLeft, ChevronRight } from '@material-ui/icons';
+
+
+import { useHistory } from 'react-router';
+import { FunctionComponent, useState } from 'react';
```

Comment:
Import statements from packages should be on top of the file

Reply:
Thanks for your review. Working on it